src/specimens/Html.js

```diff
@@ -77,27 +120,50 @@ class Html extends React.Component {
   constructor() {
     super();
     this.state = {
-      viewSource: false
+      viewSource: false,
+      parentWidth: 0,
+      screenSize: SCREENSIZES[0]
```

Comment:
Setting `screenSize` here from the default `SCREENSIZES` won't work if you have customized them.